site/installation/README.md

```diff
@@ -95,3 +97,4 @@ kpt help
 [cloud-sdk]: https://github.com/GoogleCloudPlatform/cloud-sdk-docker
 [linux]: https://github.com/GoogleContainerTools/kpt/releases/download/v1.0.0-alpha.4/kpt_linux_amd64
 [darwin]: https://github.com/GoogleContainerTools/kpt/releases/download/v1.0.0-alpha.4/kpt_darwin_amd64
+[migration guide]: installation/migration/01-migration.md
```

Comment:
you can move to /docs and then link using

https://github.com/GoogleContainerTools/kpt/blob/next/docs/migration.md